### ISSUE TITLE
[Feature Request] Ability to hide specific handlers

### ISSUE THREAD

**Issue by H-QueerCoded** [GTNewHorizons/NotEnoughItems#180]

**Comment by MagmaBro123** [comment]
This would be very nice.

**Comment by calreveraster** [comment]
Id also like to throw my support in the ring for this one - in 1.12 we have it with crafttweaker/jei, and I'd really love to see that in 1.7.10

**Comment by slprime** [comment]
what problem is this supposed to solve? give an example please

**Comment by slprime** [comment]
https://github.com/GTNewHorizons/NotEnoughItems/issues/406 some issue

**Comment by calreveraster** [comment]
> what problem is this supposed to solve? give an example please

In a heavily modded pack, many pack makers prefer to decide what items are, and are not craftable. 

Certain mods (such as Thermal Dynamics) create new crafting groups for certain things (such as cover crafting). In the case of my modpack, I disable covers, remove the recipes and hide them as we ship the pack with a pre-made block.properties file in a shader so that a fully-integrated shader is available to the player out of the box. 

In the case of the described Thermal Dynamics example, even if you disable all recipes for cover crafting and remove covers from NEI using either TD's configs or minetweaker, the handler still continues to persist. In those cases, the ability to hide the handler rounds out the cleanliness of making such modifications to the pack. 

In other examples, sometimes a pack has one too many crafting tables and you may want to consolidate, which would eliminate certain handlers from being utilized.

tl;dr handlers persist sometimes even if you remove all recipes associated with a handler, so the handler would need to be manually hidden in these instances. The ability to do so would be a huge help to pack makers that desire the ability to make their pack as clean and seamless as possible for the player.
